.github/workflows/run-validation.yml

```diff
@@ -59,10 +59,45 @@ jobs:
         shell: bash
         run: az containerapp delete -n ${{ env.TEST_CONTAINER_APP_NAME }} -g ${{ vars.TEST_RESOURCE_GROUP_NAME }} -y
 
-      - name: Delete pushed image

```

Comment:
Isn't this pushed image deletion still required for the previous test? Wouldn't we leave an image in the container registry there?

Reply:
OK I see that we removed that logic in multiple places. Could you please share more details on why we don't need this step anymore?

Reply:
Since the image is created by the CLI, we would not be able to delete the image from our gha

Reply:
Thanks, so do we know what are the consequences? Is it just going to add new images forever in the ACR used for the tests? If yes we should consider fixing it after this PR, maybe by deleting the whole ACR instance and recreating from scratch every time we start new tests?